ci/functional/job_cleanup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Common cleanup for running after a functional test
+
+set -uex
+
+rm -rf install/lib/daos/TESTING/ftest/avocado/job-results/job-*/html/
+
+# Remove the latest avocado symlink directory to avoid inclusion in the
+# jenkins build artifacts
+unlink install/lib/daos/TESTING/ftest/avocado/job-results/latest
+rm -rf "Functional/"
+mkdir "Functional/"
+# compress those potentially huge DAOS logs
+if daos_logs=$(find \
+    install/lib/daos/TESTING/ftest/avocado/job-results/job-*/daos_logs/* \
+    -maxdepth 0 -type f -size +1M); then
+  lbzip2 $daos_logs
+fi
+  arts="$arts$(ls *daos{,_agent}.log* 2>/dev/null)" && arts="$arts"$'\n'
+  arts="$arts$(ls -d \
+        install/lib/daos/TESTING/ftest/avocado/job-results/job-* 2>/dev/null)" \
+        && arts="$arts"$'\n'
+  if [ -n "$arts" ]; then
+    mv $(echo $arts | tr '\n' ' ') "Functional/"
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Double quote to prevent globbing and word splitting. [SC2086]